### PR TITLE
bump version and add dep

### PR DESCRIPTION
**Pull request by jackgerrits**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -19,6 +20,9 @@ source:
   - url: https://github.com/gabime/spdlog/archive/eb32206.tar.gz
     sha256: c60cd3742d24c3f8014618ea7e54c7fa3d5fb3886cfd59281101e6f5058dfcc5
     folder: ext_libs/spdlog
+  - url: https://github.com/boostorg/math/archive/88c4b8d.tar.gz
+    sha256: d07fb5deacc06c3a7a481c628dfee6ce920ff31c0101e3eca96311969d6d96f5
+    folder: ext_libs/boost_math
```
Comment (ngam) [inline recipe/meta.yaml:25]: Would you be interested in adding this as a conda package @jackgerrits?

Comment (ngam) [reply recipe/meta.yaml:25]: I see we can tackle this later then.

**Review by ngam — APPROVED** [review]
This looks good to me; feel free to merge whenever you're ready :)

**Comment by ngam** [comment]
@conda-forge-admin, please rerender

**Comment by ngam** [comment]
@conda-forge-admin, please rerender

**Comment by jackgerrits** [comment]
Thanks @ngam !

**Pull request merged**